application/__tests__/example.macro.integration.js

```diff
@@ -2,12 +2,23 @@
 
 const { integrationTestPort } = require('../../constants')
 
-describe('Localhost', () => {
+describe('Example page rendering', () => {
   beforeEach(async () => {
-    await page.goto(`http://localhost:${integrationTestPort}`)
+    await page.goto(`http://localhost:${integrationTestPort}/design-library/ask-for-consent/`)
```

Comment:
Should we do a full:

```
const baseUrl=`http://localhost:${integrationTestPort}`
```

That way we can use:

```
await page.goto(`${baseUrl}/design-library/ask-for-consent/`)
```

Or we could wrap it all up in:

```
const visit = url => page.goto(`http://localhost:${integrationTestPort}${url}`)

await visit('/design-library/ask-for-consent/`)
```

Reply:
👍